js/background.js

```diff
@@ -2,6 +2,8 @@
 //We can't populate it later, as selected tabs get deselected on a click inside a tab.
 var tabsToSave = [];
 
+var syncEnabled=true;//DEBUG - TODO REMOVE OR FALLBACK GRACEFULLY
```

Comment:
I don't know if there is a need for the sync mechanism to be togglable, do you have something specific in mind ? 
The option I suggest is that for now sync can't be disabled, and if we receive feedback from users that the storage gets filled too quickly, we add the ability to disable sync storage - thus falling back to the bigger local storage.

```suggestion
var syncEnabled=true;//This variable controls whether to use the chrome sync storage, along with its size limitations, or the much larger chrome local storage. The option is currently not exposed to the users.
```

Reply:
Yeah, it's totally fine. Let it be as an "emergency" switch